src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,10 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        int winTeamNumber = result1 > result2 ? 1 : result1 < result2 ? 2 : 0;
+        int myPrediction = bet1 > bet2 ? 1 : bet1 < bet2 ? 2 : 0;
+        return (result1 + "" + result2).equals(bet1 + "" + bet2) ? 2 : winTeamNumber == myPrediction ? 1 : 0;
```

Comment:
it's too complicated, try to simplify this solution

Reply:
do not push before mvn clean package shows "Success"

Reply:
done